### DISCUSSION TITLE
Reseller Setup Helps

### DISCUSSION THREAD

**Discussion by dog199200** [composer/composer#12199] [Q&A / Support]
So I have a reseller host, and i've been working with my server host to get some dependencies like ffmpeg (https://github.com/PHP-FFMpeg/PHP-FFMpeg) installed. The issue is how do we get it so that cPanel/WHM can see the extensions installed by composer? As it stands now, nothing installed by composer can be accessed via any client on the hosting. If you can point me in the direction needed so I can let the host know exactly what needs to be done to get composer apps to be see via clients i'd appreciate it.

Server info: php 8.1, 8.2, and 8.3
Composer and all dependencies were installed today and up to date.

**Comment by Seldaek** [comment]
Sorry but I'm really not sure how to help you here. I have no clue how cPanel works, and no clue what WHM is.. Please reach out to these for assistance as they'll most likely be more able to help. We cannot support every project's Composer integration if they don't do things in a work-out-of-the-box way..
